etc/sysconfig/ceph

```diff
@@ -24,3 +24,11 @@ TCMALLOC_MAX_TOTAL_THREAD_CACHE_BYTES=134217728
 # currently active on the node.
 #
 CEPH_AUTO_RESTART_ON_UPGRADE=no
+
+## unmap all devices, or only those configured in /etc/ceph/rbdmap
+#
+# By default, "rbdmap unmap" (systemctl stop rbdmap) unmounts and unmaps
+# all locally mapped RBD images. Configuring "no" here overrides this
+# behaviour so that only RBD images specified in /etc/ceph/rbdmap are
+# unmounted and unmapped.
+RBDMAP_UNMAP_ALL=yes
```

Comment:
It's probably a good question to raise to the ceph-users mailing list. My assumption is that most users that use this script use the associated service/systemd unit file that maps on startup and unmaps on shutdown. In that case, much like mapping file systems specified in /etc/fstab on startup and unmapping all mounted file systems on shutdown, I would expect this script, in the shutdown case to unmap *all* mapped RBD devices.